baseplate/context/memcache/lib.py

```diff
@@ -191,22 +199,25 @@ def decompress_and_unpickle(key, serialized, flags):
 
 
 def make_pickle_and_compress_fn(min_compress_length=0, compress_level=1):
-    """Create a serialization method compatible with decompress_and_unpickle().
+    """Create a legacy serializer.
```

Comment:
This looks like the older counterpart tp the "modern" stuff I asked about above. What constitutes legacy vs modern? Is it work linking to an explanation, or are these familiar concepts for better-versed memcache users?